packages/typespec-client-generator-core/test/types/multipart-types.test.ts

```diff
@@ -61,6 +61,22 @@ describe("typespec-client-generator-core: multipart types", () => {
       code: "@azure-tools/typespec-client-generator-core/conflicting-multipart-model-usage",
     });
   });
+  it("multipart conflicting model usage for only multipart operations", async function () {
+    await runner.compile(
+      `
+        @service({title: "Test Service"}) namespace TestService;
+
+        model MultiPartRequest {
+          id: string;
+          profileImage: bytes;
+        }
```

Comment:
Should we add an `Address` model and verify usage of it is JSON and not MultipartFormData?

Also suggest we can verify the `MultiPartRequest` model be MultipartFormData and not JSON.

BTW, why to have a PUT and a POST in op?

Reply:
_Should we add an Address model and verify usage of it is JSON and not MultipartFormData?
Also suggest we can verify the MultiPartRequest model be MultipartFormData and not JSON._

-> Updated test. And I also add another test for mixed operation scenarios

Reply:
_BTW, why to have a PUT and a POST in op?_
-> With different method, there is no need to add route to distinguish them. Maybe it caused confusion, I have updated the test.